lib/commands.js

```diff
@@ -20,15 +20,18 @@ function hasAdminPermissions (member) {
 
 function noPermissions (message) {
   const errorMessage = message.member
-    ? 'You do not have enough permissions to run this command'
-    : 'You need to be in guild channel for this command to work'
+    ? 'You can\'t run this command '
+    : 'You need to be in a guild channel for this command to work'
 
-  return message.channel.send({
+  const place = message.channel && message.member ? 'inside ' + message.channel : ''
```

Comment:
use here instead of ''.

Reply:
This will add `here` to the guild message, which seems wrong?

Reply:
No, that is fine

Reply:
Just do it :d